### PR TITLE
Tap server reformat

### PR DESCRIPTION
**Pull request by at88mph**
I'm sorry this is a really big PR.  There is some repetition with the server libraries.

The `cadc-tap-[mysql|pg|oracle|sybase]` libraries contain sample SQL per database for setting up a TAP schema.

The `cadc-tap-server-[mysql|pg|oracle|sybase]` libraries contain the necessary library files to connect to those specific databases, along with an `examples` folder as a started.

### CODE REVIEW

**Comment by pdowler** [comment]
My intention is to eventually have the cadc-tap-schema library be able to create and maintain the tap-schema tables and content... I'm not sure how those cadc-tap-{db} modules would be used for that. I suspect that content belongs in the cadc-tap-server-{db} modules for now. 

I haven't yet look at the actual content, but the extra N modules with those names look like they will be more confusing instead of less.

**Comment by at88mph** [comment]
They only contain the necessary SQL that cadc-tap-schema would probably be creating.  Should the SQL be migrated into cadc-tap-schema then?


Dustin N. Jenkins
Tel/Tél: 250.363.3101 | Mobile: 250.217.4970 | Facsimile/télécopieur: 250.363.0045 | Dustin.Jenkins@nrc-cnrc.gc.ca<mailto:Dustin.Jenkins@nrc-cnrc.gc.ca>
National Research Council Canada | 5071 West Saanich Rd, Victoria BC. V9E 2E7
Conseil national de recherches Canada | 5071, ch. West Saanich, Victoria (C.-B) V9E 2E7
Government of Canada | Gouvernement du Canada








On May 22, 2018, at 14:45, Patrick Dowler <notifications@github.com<mailto:notifications@github.com>> wrote:


My intention is to eventually have the cadc-tap-schema library be able to create and maintain the tap-schema tables and content... I'm not sure how those cadc-tap-{db} modules would be used for that. I suspect that content belongs in the cadc-tap-server-{db} modules for now.

I haven't yet look at the actual content, but the extra N modules with those names look like they will be more confusing instead of less.

**Comment by at88mph** [comment]
Builds passing now.

**Comment by at88mph** [comment]
Done.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+before_script: 
+  - openssl s_client -CApath /etc/ssl/certs/ -connect plugins.gradle.org:443 </dev/null | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > /tmp/gradle.crt; sudo keytool -importcert -noprompt -file /tmp/gradle.crt -trustcacerts -keystore $JAVA_HOME/jre/lib/security/cacerts -alias root -storepass changeit;
 
-script: for mod in cadc-tap-schema cadc-jsqlparser-compat cadc-tap-server cadc-adql cadc-test-tap example-tap; do cd $mod; gradle --stacktrace assemble javadoc install || break -1; cd ..; done
+script: 
+  - cd cadc-tap-server; gradle --stacktrace clean assemble javadoc test install || break -1; cd ../
+  - for db in pg oracle; do cd cadc-tap-server-$db; gradle --stacktrace clean assemble javadoc test install; cd ../; done
+  - for mod in cadc-tap-schema cadc-jsqlparser-compat cadc-adql cadc-test-tap example-tap; do cd $mod; gradle --stacktrace assemble javadoc install || break -1; cd ..; done
```
Comment (pdowler) [inline .travis.yml]: cadc-tap-schema needs to be built before cadc-tap-server (dependency)

cadc-jsqlparser-compat cadc-adql are plugins so after is correct

**Comment by at88mph** [comment]
So that’s what is happening.  Thanks.

**Pull request merged**